src/lib/autoinstall/clients/inst_autosetup_upgrade.rb

```diff
@@ -226,6 +226,23 @@ def main
           false
         )
 
+        #
+        # Checking if at least one base product has been selected/evaluated.
+        #
+        begin
+          Product.FindBaseProducts
+        rescue StandardError
+          msg = _("No new base product has been set.\n" \
+           "Please check the <b>software</b>/<b>products</b> entry in the " \
+           "AutoYaST configuration file.<br><br>" \
+           "Following base products are available:<br>")
+          Yast::AutoinstFunctions.available_base_products_hash.each do |product|
+            msg += "#{product[:name]} (#{product[:summary]})<br>"
+          end
+          Yast::Popup.LongError(msg) # No timeout because we are stopping the upgrade.
```

Comment:
I'd prefer using the new [yast2/popup.rb](https://github.com/yast/yast-yast2/blob/master/library/general/src/lib/yast2/popup.rb) library for popups.